### DISCUSSION TITLE
How to disable diagnostics highlighting?

### DISCUSSION THREAD

**Discussion by poperigby** [prabirshrestha/vim-lsp#1165] [Q&A] [answered]
I just want the diagnostics to show up in the gutter and not highlight any text. Is that possible?

**Comment by prabirshrestha** [comment, marked as answer]
should be document in doc. Please let us know if we can improve docs.

`:h vim-lsp`.

**Reply by poperigby** [reply]
I looked through the documentation and was turning different things off, but I couldn't figure it out. Could you tell me the name of the feature please? I can probably figure it out from there.

**Reply by prabirshrestha** [reply]
once you open the documentation you can use normal vim search i.e. `/diagnostics` and see what are there. There is table of contents in the doc.

Another option is to look in this file. https://github.com/prabirshrestha/vim-lsp/blob/1f98a0656620bfc03dcb8bad87aa76a6304b3b73/plugin/lsp.vim#L17-L37

Over time I have tried to group features together or rename some of the settings so it makes better sense.

**Comment by juque** [comment]
```
call lsp#disable_diagnostics_for_buffer()
```
